javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -510,6 +510,8 @@ private static boolean sameAnnotationValueValue(Object val1, Object val2) {
                 }
             }
             return true;
+        } else if ((val1 instanceof AnnotationMirror) && (val2 instanceof AnnotationMirror)) {
+            return areSame((AnnotationMirror) val1, (AnnotationMirror) val2);
         } else if ((val1 instanceof AnnotationValue) && (val2 instanceof AnnotationValue)) {
```

Comment:
@mernst I'm not sure when/if an AnnotationValue's value is an AnnotationValue.  Do you know?

Reply:
Here is an example from the [JLS](https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.7.3):
```@Author(@Name(first = "Joe", last = "Hacker"))```